apps/docs/src/styles.scss

```diff
@@ -1,8 +1,22 @@
 @use '../../../libs/anglify/src/styles/index';
-@use '../../../libs/anglify/src/styles/themes/light';
+@use '../../../libs/anglify/src/styles/themes/light' as *;
+@use '../../../libs/anglify/src/styles/themes/dark' as *;
 @use '../../../libs/anglify/src/modules/table';
 @use '../../../libs/anglify/src/styles/settings/typography.scss';
-/* You can add global styles to this file, and also import other style files */
+
+:root {
+  @include light-theme;
+
+  &.light {
+    @import '~highlight.js/scss/github.scss';
+  }
+
+  &.dark {
+    @import '~highlight.js/scss/github-dark.scss';
+    @include dark-theme;
+  }
+}
+
```

Comment:
The imports in lines 11 & 15 should not be necessary:

```scss
@use '../../../libs/anglify/src/styles/index';
@use '../../../libs/anglify/src/styles/themes/light' as *;
@use '../../../libs/anglify/src/styles/themes/dark' as *;
@use '../../../libs/anglify/src/modules/table';
@use '../../../libs/anglify/src/styles/settings/typography.scss';

:root {
  @include light-theme;

  &.dark {
    @include dark-theme;
  }
}
```

Reply:
Those were added by yourself, and they are necessary, I think. I did not modify those.

Reply:
I am as dumb as a brick. I thought these were Anglify Style Imports 🥴

My bad. Thank you for questioning this